backend/btrixcloud/models.py

```diff
@@ -88,6 +88,14 @@ class User(BaseModel):
     invites: Dict[str, InvitePending] = {}
     hashed_password: str
 
+    # Consecutive failed logins, reset to 0 on successful login.
+    failed_logins: Optional[int] = 0
+
+    # If user is logged, they are unable to log in until they reset their
+    # password from the /forgot-password email or /users/me/password-change.
+    # This is to prevent brute force cracking of passwords via /login.
+    locked: Optional[bool] = False
```

Comment:
Hm, do we actually need a separate `locked` field?

We can just check the `failed_logins > 5` and consider that to be locked.

Completing the reset flow would then set it to 0.

Reply:
It was so that we could reset the failed logins when the password reset email was sent, but if we do that on resetting password instead then yeah, not needed! Will change.